OpenTabletDriver.Desktop/Binding/BindingHandler.cs

```diff
@@ -109,6 +115,32 @@ private void HandleAuxiliaryReport(IAuxReport report)
             HandleBindingCollection(report, AuxButtons, report.AuxButtons);
         }
 
+        private void HandleAbsoluteWheelReport(IAbsoluteWheelReport report)
+        {
+            var wheelMovement = report.WheelPosition != null && _lastWheelPosition != null;
+
+            bool clockWise = false;
+            bool antiClockwise = false;
+            if (wheelMovement && _device.Configuration.Specifications.Wheel != null)
+            {
+                var steps = _device.Configuration.Specifications.Wheel.StepCount;
+                var halfSteps = steps / 2;
+                var treeHalf = halfSteps * 3;
+
+                var movement = (report.WheelPosition - _lastWheelPosition + treeHalf) % steps - halfSteps;
```

Comment:
```suggestion
                var tripleHalfSteps = (steps / 2) * 3;

                var movement = (report.WheelPosition - _lastWheelPosition + tripleHalfSteps) % (steps / 2);
```

Bit clearer imo (how I'd do it, feel free to disregard), also saves making 2 variables. Also, `steps - halfSteps` is just halfSteps because "x - 1/2 x" is just... 1/2 x.

Reply:
Its `(math % steps) - halfSteps`, not `math % (steps - halfSteps)`